src/styles/blocks/interesting-articles.css

```
@@ -41,6 +41,33 @@
   color: red;
 }
 
+.interesting-articles__add-article {
+  padding: 12px;
+  font-weight: 500;
+  border: 1px solid black;
+  background-color: #fcffe7;
+  margin: 40px 0;
+  width: 33%;
+}
+
+.interesting-articles__article-preview,
+.interesting-articles__form {
+  text-align: left;
+}
+
+.interesting-articles__form {
+  width: 66%;
+  margin: 0 auto;
+}
+
+.interesting-articles__form.interesting-articles__form--hidden {
+  display: none;
+}
```

зачем тут специфичность повышается?
```suggestion
.interesting-articles__form--hidden {
  display: none;
}
```

если не работает, то сделать чтоб работало